test/test-rep.js

```diff
@@ -0,0 +1,36 @@
+var StringBuilder = require('../src/string-builder');
+var assert = require('chai').assert;
+var sinon = require('sinon');
+var jsonfile = require('jsonfile');
+
+var file = './fixtures/data.json';
+var fixtures = jsonfile.readFileSync(file);
+
+describe('StringBuilder #rep', function() {
+    it('Should add twice the paramaters', function() {
+        var sb = new StringBuilder();
+
+        sb.rep('hello','world', 2);
```

Comment:
can you tests how it behaves when the `args` are not just `strings` but anything the `cat` would accept? (arrays, functions, numbers, etc)